sicp-doc/sicp-pict.scrbl

```diff
@@ -222,10 +303,13 @@ segments (w.r.t. the unit square).}
 Constructs a painter that draws a stick figure given by the
 vects (w.r.t. the unit square).}
 
-@defproc[(procedure->painter [f procedure?]) painter/c]{
+@defproc[(procedure->painter [f (-> (real-in 0 1)
+                                    (real-in 0 1)
+                                    (or/c real? string? color-object?))]
+                             [size real? 100]) painter/c]{
```

Comment:
Can we find a name for 
    (-> (real-in 0 1)
          (real-in 0 1)
          (or/c real? string? color-object?))
?

Consider a beginner that starts with SICP and doesn't know Racket that well.
It would be simpler to see something like:

  procedure->painter :   some-name-here -> painter/c
  
  Perhaps `painter-procedure` ?

Reply:
This sounds good to me.